packages/engine/src/physics/components/VelocityComponent.ts

```diff
@@ -9,13 +9,15 @@ import { createMappedComponent } from '../../ecs/functions/ComponentFunctions'
  */
 
 export type VelocityComponentType = {
-  velocity: Vector3
+  linearVelocity: Vector3
```

Comment:
we should rename these to `linear` and `angular` as the velocity word is redundant, makes it easier to read